go.mod

```diff
@@ -5,14 +5,13 @@ go 1.14
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/alessio/shellescape v1.4.1
-	github.com/evanphx/json-patch/v5 v5.2.0
-	github.com/mattn/go-isatty v0.0.12
-	github.com/pelletier/go-toml v1.8.1
+	github.com/evanphx/json-patch/v5 v5.5.0
+	github.com/mattn/go-isatty v0.0.13
+	github.com/pelletier/go-toml v1.9.3
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.1.3
```

Comment:
v1.2.0 just released per #k8s-code-organization but looking at the release notes require some changes to auto-completion, will follow up with that as the PR is still good and already tested / minimal changes required.